src/SIM/SIMbase.C

```diff
@@ -2544,9 +2543,8 @@ bool SIMbase::injectPatchSolution (Vector& sol, const Vector& vec,
       nndof != getNoFields(basis) && this->getNoFields(2) > 0) {
```

Comment:
Skulle det ha stått "basis" i stedet for 2 i det siste getNoFields-kallet her? Eller er det noe finurlig logikk som gjør at det blir sånn.  Dessuten this-> også på det første kallet. Forøvrig det samme i extractPatchSolution.